### PR TITLE
Set up postgres and redis seeding files, fixed routes for both databases.

### PR DESCRIPTION
**Pull request by cdag22**

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -99,8 +102,11 @@ class App extends React.Component {
     const { dataset } = e.target.tagName === 'DIV' ? e.target.children[0] : e.target;
     const { images } = this.state;
     const fullImg = images.find(img => img.color === dataset.color && img.orientation === dataset.orientation && img.size === 'full');
+    // e.preventDefault();
```
Comment (thejivecat) [inline client/components/app.jsx]: need to remove commented out code

File: db/rawData/imageText.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+23	full	flame	bottom	https://s3-us-west-2.amazonaws.com/fec-rei/flame/bottom_full.jpg
+24	full	flame	front	https://s3-us-west-2.amazonaws.com/fec-rei/flame/front_full.jpg
+25	full	flame	inside	https://s3-us-west-2.amazonaws.com/fec-rei/flame/inside_full.jpg
+26	full	flame	inuse_1	https://s3-us-west-2.amazonaws.com/fec-rei/flame/inuse_1_full.jpg
+27	full	flame	inuse_2	https://s3-us-west-2.amazonaws.com/fec-rei/flame/inuse_2_full.jpg
+28	full	flame	inuse_3	https://s3-us-west-2.amazonaws.com/fec-rei/flame/inuse_3_full.jpg
+29	full	flame	outside	https://s3-us-west-2.amazonaws.com/fec-rei/flame/outside_full.jpg
+30	full	flame	top	https://s3-us-west-2.amazonaws.com/fec-rei/flame/top_full.jpg
+31	select	kiwi	 	https://www.rei.com/media/f85ca8b1-5e82-44d6-97c1-cf38c88154b0
+32	select	flame	 	https://rei.com/media/4f72b233-00f4-449b-b3c8-c338b526cb05?size=60`
```
Comment (thejivecat) [inline db/rawData/imageText.js]: need newline here

File: db/rawData/productsText.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+92	Vapor V Climbing Shoes	Garmont	475806	Blue	188.69	5.0	45
+93	Street Climbing Shoes	Garmont	325102	Khaki	112.86	4.6	195
+94	TC Pro Climbing Shoes	Arc'teryx	392989	Black	57.82	2.7	122
+95	Street LV Climbing Shoes	evolv	555844	Khaki	13.10	4.8	129
+96	Street LV Climbing Shoes	Black Diamond	609732	Blue	140.04	2.4	109
+97	Vapor V Climbing Shoes	La Sportiva	739575	Kiwi	42.68	4.4	16
+98	Techno X Climbing Shoes	evolv	632731	Blue	199.10	4.4	221
+99	Origin Climbing Shoes	evolv	559390	Kiwi	193.59	2.6	95
+100	Origin Climbing Shoes	Butora	254477	Khaki	52.82	1.6	82
+101	Oxygym Climbing Shoes	Arc'teryx	798357	Khaki	24.92	5	234`;
```
Comment (thejivecat) [inline db/rawData/productsText.js]: need newline here

File: db/redis/dbConfig.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+//
+require('dotenv').config();
+const redis = require('redis');
+const process = require('process');
+
+const host = process.env.REDIS_HOST;
+const port = process.env.REDIS_PORT;
+const client = redis.createClient({ port, host });
+
+module.export = client;
```
Comment (thejivecat) [inline db/redis/dbConfig.js]: need newline here

File: db/redis/redisDataGenerator.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+function buildResp(name, data, startIndex, wstream) {
+  const keys = Object.keys(data[0]);
+  return data.forEach((item, i) => {
+    let line = `*${2 + keys.length * 2}\r\n$5\r\nHMSET\r\n$${name.length + 1 + String(i + startIndex + 1).length}\r\n${name}:${i + startIndex + 1}\r\n` + keys.map(k => `$${k.length}\r\n${k}\r\n$${String(item[k]).length}\r\n${item[k]}\r\n`).join('')
+    wstream.write(line + '\r\n');
+  });
+}
+
+
+buildFile(FILE_NAME, data[FILE_NAME])
```
Comment (thejivecat) [inline db/redis/redisDataGenerator.js]: need newline

File: db/seedingSetup/modifyImageFiles.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    });
+  });
+};
+
+getImageFiles().then(data => {
+  fs.writeFile(path.join(__dirname, 'imageList.js'), JSON.stringify(data), (error) => {
+    if (error) console.log(error);
+    else console.log('success');
+  });
+});
```
Comment (thejivecat) [inline db/seedingSetup/modifyImageFiles.js]: newline here

File: db/sql/dbConfig.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+// Setup Database
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+module.exports = database;
```
Comment (thejivecat) [inline db/sql/dbConfig.js]: newline here

File: unsplashApiKey.js [hunk 8]
```diff
@@ -0,0 +1 @@
+module.exports = '10258406acbe31f39428b5adc34654cd2d0da60405b969451a20587c98eaedea';
```
Comment (thejivecat) [inline unsplashApiKey.js:1]: gitignore this

File: knexfile.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
+const Path = require('path');
```
Comment (thejivecat) [inline knexfile.js:1]: take out personal login info

**Review by thejivecat — CHANGES_REQUESTED** [review]
change stuff I've commented and you are good

File: client/components/app.jsx (hunk 1 above)
Comment (cdag22) [reply client/components/app.jsx]: thanks. ill put more variables in .env and .gitignore my unsplash api key

**Review by cchasewood — COMMENTED** [review]
Looks good for the most part, I'm noticing a serious lack of one-liners though

**Pull request merged**